### PR TITLE
Web: renderer for TypeSpec + React rendering fixes

### PR DESCRIPTION
**Pull request by abock**

### CODE REVIEW

File: Clients/Xamarin.Interactive.Client.Web/ClientApp/renderers/TypeSystemRenderers.tsx [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+            return 'ulong'
+        case 'Single':
+            return 'float'
+        case 'nint':
+            return 'nint'
+        case 'nuint':
+            return 'nuint'
+        case 'nfloat':
+            return 'nfloat'
+    }
```
Comment (sandyarmstrong) [inline Clients/Xamarin.Interactive.Client.Web/ClientApp/renderers/TypeSystemRenderers.tsx]: do you want a `default: return typeName.name`?

Comment (sandyarmstrong) [reply Clients/Xamarin.Interactive.Client.Web/ClientApp/renderers/TypeSystemRenderers.tsx]: nevermind I shouldn't read things upside down.

**Comment by bojanrajkovic** [comment]
I have no idea how I managed to hit request changes there.

**Comment by abock** [comment]
https://twitter.com/abock/status/992436794225512448

**Pull request merged**